synfig-studio/src/gui/pluginmanager.cpp

```diff
@@ -179,7 +179,8 @@ studio::PluginManager::load_plugin( const std::string &file, const std::string &
 {
 	synfig::info("   Loading plugin: %s", etl::basename(plugindir).c_str());
 
-	std::string id = file;
+	static int plugin_count = 0;
+	const std::string id = "plugin" + std::to_string(++plugin_count);
```

Comment:
I thought you'd use plugin name + count :)

Reply:
Yes. But in this case, we have to set plugin naming rules (no spaces, only Latin characters, etc.). 
So I think this will be better solution.

Reply:
Got it. Approved